magento-research/pwa-studio
# (#1332) Add timeout on tooltip to hide automatically

## Description
Added timeout on the Tooltips so that they automatically disappear after 2.5s

## Related Issue





Closes #1332 .

## Verification Steps

1. Go to the PDP.
2. Click on a swatch (e.g. colour or size).
3. Tooltip should appear. Tooltip will disappear after 2.5s

## How have YOU tested this?



Checked the timeout in Chrome 75 on MacOS.

## Screenshots / Screen Captures (if appropriate):

## Proposed Labels for Change Type/Package




## Checklist:
